src/theme.js

```diff
@@ -0,0 +1,112 @@
+import theme from "@rebass/preset"
+
+const customTheme = {
+  ...theme,
+  breakpoints: [320, 768, 1024, 1440],
+  colors: {
+    darkBlue: "#0C1D60",
+    lightBlue: "#7BD5F5",
+    gold: "#F7D663",
+    white: "#FFF",
+  },
+  fonts: {
+    body: "system-ui, sans-serif",
+    heading: "inherit",
+    monospace: "Menlo, monospace",
+  },
+  fontSizes: [
+    "0.75rem",
+    "1rem",
+    "1.333rem",
+    "1.777rem",
+    "2.369rem",
+    "3.157rem",
+    "4.209rem",
+    "5.61rem",
+    "7.478rem",
+    "9.969rem",
+    "13.288rem",
+    "17.713rem",
+    "23.612rem",
+    "31.475rem",
+    "41.956rem",
+    "55.927rem",
+    "74.551rem",
+    "99.376rem",
+  ],
+  fontWeights: {
```

Comment:
What do you think about just following the `body` (400) and `heading` (900) conventions [from the official Rebass preset](https://github.com/rebassjs/rebass/blob/master/packages/preset/src/index.js) and [the Material preset](https://github.com/rebassjs/rebass/blob/master/packages/preset-material/src/index.js)? That way we can easily map font weights to their corresponding line heights.

Also, we won't really need be needing the other weights since I've designed the type system around just those two anyway. 😄

Reply:
doesn't matter really if it is 700 or 900 since we are just using 1 font for bold text